avm/res/synapse/workspace/main.bicep

```diff
@@ -352,7 +352,7 @@ module workspace_firewallRules 'firewall-rules/main.bicep' = [
 ]
 
 // Endpoints
-module workspace_privateEndpoints 'br/public:avm/res/network/private-endpoint:0.7.1' = [
+module workspace_privateEndpoints 'br/public:avm/res/network/private-endpoint:0.9.0' = [
   for (privateEndpoint, index) in (privateEndpoints ?? []): {
     name: '${uniqueString(deployment().name, location)}-workspace-PrivateEndpoint-${index}'
     scope: resourceGroup(privateEndpoint.?resourceGroupName ?? '')
```

Comment:
Must be updated to
```bicep
scope: !empty(privateEndpoint.?resourceGroupResourceId)
? resourceGroup(
    split((privateEndpoint.?resourceGroupResourceId ?? '//'), '/')[2],
    split((privateEndpoint.?resourceGroupResourceId ?? '////'), '/')[4]
  )
: resourceGroup(
    split((privateEndpoint.?subnetResourceId ?? '//'), '/')[2],
    split((privateEndpoint.?subnetResourceId ?? '////'), '/')[4]
  )
```
as per the common-types v 0.5.1.

Source: https://azure.github.io/Azure-Verified-Modules/specs/bcp/res/interfaces/#private-endpoints

Reply:
done